### ISSUE TITLE
[Suggestion] Reporting users

### ISSUE THREAD

**Issue by Kaamkiya** [discuitnet/discuit#134]
We should be able to bring users who are breaking the rules to admins' attention. 

Right now, we can report comments and posts, but sometimes the comments aren't against the rules, just rude. If we could report accounts, it would be easier to say that someone is generally a pain/liar and should be removed.

**Comment by previnder** [comment]
Good idea. Right now, the only way to report accounts is to alert the admins on Discord. The admins ideally should be notified in some way when these user account reports come in.
